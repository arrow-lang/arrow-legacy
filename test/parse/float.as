32.23;
52e+30;
2.0;
3.251e30;
3.251e-30;
