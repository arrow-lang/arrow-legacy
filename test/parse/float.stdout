<?xml version="1.0" encoding="utf-8"?>
<Module>
 <Float>32.23</Float>
 <Float>52e+30</Float>
 <Float>2.0</Float>
 <Float>3.251e30</Float>
 <Float>3.251e-30</Float>
</Module>
